src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractValidationServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import static org.junit.Assert.*;
+
+import java.sql.Date;
+import java.util.Calendar;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.batch.businessobject.ConcurStandardAccountingExtractFile;
+import edu.cornell.kfs.concur.batch.fixture.ConcurStandardAccountingExtractFileFixture;
+import edu.cornell.kfs.concur.batch.service.ConcurStandardAccountingExtractValidationService;
+
+public class ConcurStandardAccountingExtractValidationServiceImplTest {
+    
```

Comment:
should probably add some new tests for the new validateConcurStandardAccountExtractFile method